src/main/java/org/sonarsource/sonarlint/ls/connected/ProjectBindingManager.java

```diff
@@ -307,21 +313,28 @@ public void updateAllBindings() {
     // Clear cached bindings to force rebind during next analysis
     folderBindingCache.clear();
     fileBindingCache.clear();
-
+    List<String> failedServerIds = new ArrayList<>();
     // Start by updating all engines that are already started and cached
-    connectedEngineCacheByServerId.entrySet().forEach(entry -> {
-      String serverId = entry.getKey();
+    connectedEngineCacheByServerId.forEach((serverId, value) -> {
       ServerConfiguration serverConfiguration = createServerConfiguration(serverId);
       if (serverConfiguration != null) {
-        entry.getValue().ifPresent(engine -> updateGlobalStorageAndLogResults(serverConfiguration, engine));
+        try {
+          value.ifPresent(engine -> updateGlobalStorageAndLogResults(serverConfiguration, engine, failedServerIds, serverId));
+        } catch (Exception e) {
+
```

Comment:
Empty catch ?

Reply:
Removed. I left try-catch at the second call place because engine.getGlobalStorageStatus() may throw an exception.